content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -26,6 +26,7 @@ For information how to create a cluster with kubeadm once you have performed thi
   - Fedora 25+
   - HypriotOS v1.0.1+
   - Flatcar Container Linux (tested with 2512.3.0)
+  - Amazon Linux 2+
```

Comment:
i think we should replace:

```
* One or more machines running one of:
  - Ubuntu 16.04+
  - Debian 9+
  - CentOS 7+
  - Red Hat Enterprise Linux (RHEL) 7+
  - Fedora 25+
  - HypriotOS v1.0.1+
  - Flatcar Container Linux (tested with 2512.3.0)
  - Amazon Linux 2+
```

with:

```
* A compatible Linux host. The Kubernetes project does not maintain a matrix of supported Linux distributions.
```

Reply:
Nice. How about adding this sentence as well:

However, we do offer generic instructions that apply to Linux distributions based on Debian, Red Hat, and other Linux ecosystems.

Or something like that. Thoughts?

Reply:
> * A compatible Linux host. The Kubernetes project does not maintain a matrix of supported Linux distributions, but we do have generic instructions for distributions based on Debian, RedHat and such without a package manager.

^ SGTM

the docs team will likely have a pass on this language here too.

Reply:
I'm good with that.

Reply:
Done.